### PR TITLE
feat(web): add skip link to top navigation

### PR DESCRIPTION
**Pull request by ben-basten**
Added an invisible skip link button in the top navigation bar that appears when focused for easier keyboard navigation.

<details><summary>Screenshots</summary>
<b>Dark</b>

dark-mode

<b>Light</b>
light-mode
</details> 

## Checklist

* [x] npm run lint (linting via ESLint)
* [x] npm run format (formatting via Prettier)
* [x] npm run check:svelte (Type checking via SvelteKit)
* [x] npm test (Tests via Jest)

### CODE REVIEW

File: web/src/lib/components/shared-components/side-bar/side-bar-section.svelte [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 <section
   id="sidebar"
+  tabindex="-1"
```
Comment (ben-basten) [inline web/src/lib/components/shared-components/side-bar/side-bar-section.svelte:6]: This pulls the sidebar container out of the tab order, so the next tab stop after `<main>` is the sidebar buttons.

**Comment by alextran1502** [comment]
What is the problem you are trying to solve here?

**Comment by bo0tzz** [comment]
@alextran1502 this is for users who use screen readers and such, to streamline their navigation.

**Comment by ben-basten** [comment]
@alextran1502 I think the WCAG accessibility spec explains it better than I can - the first 2 paragraphs have a good explanation.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html

**Review by danieldietzler — APPROVED** [review]

**Pull request merged**